src/parser/jobs/mnk/modules/TwinSnakes.tsx

```diff
@@ -45,8 +46,8 @@ export class TwinSnakes extends Analyser {
 	@dependency private suggestions!: Suggestions
 
 	private history: TwinState[] = []
-	private ignoredGcds: number[] = []
-	private twinSnake?: TwinState
+	private ignoredGcds: Array<Action['id']> = []
+	private twinSnake: TwinState | undefined
```

Comment:
See also comment on Forms

Reply:
Same deal here, it's not optional, just potentially undefined.